roles/v2v.ci/tasks/miq_install_nmon.yml

```diff
@@ -24,29 +23,40 @@
         dest: "{{ nmon_local_dir }}"
   when: nmon_file_exist.stat.exists == false
 
-- name: Copy nmon to CFME machine
+# Install Nmon on CFME
```

Comment:
you already have this code(lines # 26-53) in roles/v2v.ci/tasks/rhv_install_nmon.yml 
maybe find the way to use it instead of repeat yourself.
in this way, the code will be more maintainable and readable